doc/tutorials/sagemaker/sme_deploy_model.ipynb

```diff
@@ -0,0 +1,318 @@
+{
```

Comment:
<p>The [] around TARGET_COLUMN is confusing, what is it for?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>That's in case there is a space in the name.</p>

Reply:
<p>Is the [] from JupySQL?</p>

Reply:
It is just a standard SQL syntax. If the column is called "my column" then `SELECT [my column] FROM ...` will work.

Reply:
didn't know Exasol supports that. Usually, we use "" for identifier, but ok if it works its fine.